backend/service/authn/sql.go

```diff
@@ -0,0 +1,76 @@
+package authn
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/lyft/clutch/backend/service"
+	"github.com/lyft/clutch/backend/service/db/postgres"
+)
+
+type authnToken struct {
+	userID   string
+	provider string
+
+	accessToken  []byte
+	refreshToken []byte
+	idToken      []byte
+	expiry       time.Time
+}
+
+type repository struct {
+	db *sql.DB
+}
+
+func newRepository() (*repository, error) {
+	svcName := postgres.Name
+	svc, ok := service.Registry[svcName]
+	if !ok {
+		return nil, fmt.Errorf("database '%s' not registered", svcName)
+	}
+
+	pg, ok := svc.(postgres.Client)
+	if !ok {
+		return nil, fmt.Errorf("database does not implement the required interface")
+	}
+
+	return &repository{db: pg.DB()}, nil
+}
+
```

Comment:
I see we dont' have a delete or remove function, whats our plan for token revocation?

Reply:
good question. i view revocation as the responsibility of the provider (okta in this case). when we add clutch refresh (and store the clutch-issued tokens in the db) we can add logout and revocation.